### ISSUE TITLE
usb_cdc_4g_module reports warning: USB_HCDC: The out buffer is too small

### ISSUE THREAD

**Issue by achao1104** [espressif/esp-iot-solution#150]
hello,

I just downloaded the example a couple of days ago, everything fine, but just occasionally reports warning as below:

W (119150) USB_HCDC: The out buffer is too small, the data has been lost 88

**Comment by leeebo** [comment]
Which branch and commit are you using? There is a WiFi - USB conflict bug in the current [release/v4.4](a79dc75f0a91f4b558dcefb7ae5096c71c63e2e9), please try with following commands in ESP-IDF folder:

```
git pull
git fetch --all
git cherry-pick 65f3bf4a8a21f
git submodule update -f
```

**Comment by achao1104** [comment]
yes, I am using v4.4，besides the commit below，how can I fix it in v4.4?



---Original---
From: ***@***.***&gt;
Date: Wed, Dec 22, 2021 15:09 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [espressif/esp-iot-solution] usb_cdc_4g_module reports warning: USB_HCDC: The out buffer is too small (Issue #150)




 
Which branch and commit are you using? There is a WiFi - USB conflict bug in the current release/v4.4, please try with following commands in ESP-IDF folder:
 git pull git fetch --all git cherry-pick 65f3bf4a8a21f git submodule update -f  
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by leeebo** [comment]
The stable version of iot_usbh_modem has been merged in [master](https://github.com/espressif/esp-iot-solution/tree/master/components/usb/iot_usbh_modem) and has been released as a [component in esp-registry](https://components.espressif.com/components/espressif/iot_usbh_modem), please give a try
